lib/filters/post-tags.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Filter the tags on a post to remove the default ones
+ * we just use for creating the collections
+ * @param {Array} array Array to filter
+ * @return {Array} filtered array
+ */
+module.exports = (array) => {
+  const tagsToIgnore = ['post', 'apply-for-teacher-training', 'manage-teacher-training-applications', 'register-trainee-teachers', 'find-teacher-training']
```

Comment:
This tag list differs from the `ignoredTags` in `lib/filters/all-tags.js`. Should they be the same?

Reply:
@simonwhatley I've de-duplicated these now, moving the ignored tags to a JSON config file. Thanks to @edwardhorsford  for helping!